pennylane/interfaces/__init__.py

```diff
@@ -50,5 +50,5 @@
 
     interfaces/numpy
     interfaces/torch
-    interfaces/tfe
+    interfaces/tf
```

Comment:
This is something that I realised; the code required for the 1.X 'tfe' and 2.0 'tf' interfaces is exactly the same, the only thing that differs is the import statements.

Rather than duplicating codes and tests, we should have 1 doc page, 1 test, and 1 module for the `tf` interface, but make clear to users that they must use `'tfe'` if using tf 1.0.

At some point we can quietly deprecate/remove 'tfe'.

Reply:
Nevermind, lets just remove 'tfe'.